src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,222 @@
+""" Automation of proxy provider suite
+
+:requirement: IDM-SSSD-REQ : Proxy Provider
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import subprocess
+import ldap
+from sssd.testlib.common.utils import sssdTools, \
+    SSHClient, LdapOperations
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.usefixtures('setup_sssd_krb',
+                         'create_posix_usersgroups',
+                         'sssdproxyldap',
+                         'install_nslcd',
+                         'misc_test',
+                         'netgroups')
+@pytest.mark.tier1_3
+class TestProxyMisc(object):
+    """
+    This is test case class for proxy provider suite
+    """
+    def test_bz1036758(self, multihost, backupsssdconf):
+        """
+        :title: Allow for custom attributes in RDN bz1036758
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1036758
```

Comment:
There is an issue even with the original test. The issue in this bugzilla was that the user in LDAP didn't have a DN like uid=username,ou=users,dc=example,dc=com' but `cn=Full Name,ou=users,dc=example,dc=com.` But if I see it correctly all users for the tests are created as `uid=.....`. The old versions of SSSD did had the issue that if there was only `auth_provider=ldap` but the `id_provider` was not `ldap` the DN of the user was not know and SSSD was guessing it as `uid=....`. So to properly test for this BZ a dedicated user with a DN starting e.g. with `cn=...` has to be used.

Reply:
Done